packages/server/src/controllers/gateway.controller.ts

```
@@ -66,50 +55,140 @@ export class MyGateway implements OnModuleInit {
+      logger.error('Error in handleSupervisorRequest:', error);
+      throw new ServerError('E03TA100');
+    }
+  }
+
   @SubscribeMessage('agents_request')
   async handleUserRequest(
     @MessageBody() userRequest: WebsocketAgentRequestDTO
   ): Promise<void> {
     try {
       logger.info('handleUserRequest called');
       const route = this.reflector.get('path', this.handleUserRequest);
```

ESLint

[error] 'route' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)